src/library/Uno.Material/Styles/Controls/TextBox.xaml

```diff
@@ -54,14 +60,14 @@
 												Value="{StaticResource PrimaryColor01Brush}" />
 										<Setter Target="DeleteButton.Foreground"
 												Value="{StaticResource PrimaryColor01Brush}" />
+										<Setter Target="PlaceholderElement.Foreground"
+												Value="{StaticResource PrimaryColor01Brush}" />
```

Comment:
We shouldn't use PrimaryColor01Brush, use the new ones. In this case PrimaryBrush.

Reply:
Fixed.